package.json

```diff
@@ -47,6 +47,11 @@
       "title": "Show message when a file is ignored",
       "type": "boolean",
       "default": false
+    },
+    "attemptFix": {
+      "title": "Attempt to fix errors using stylelint --fix flag (experimental)",
```

Comment:
This should be the `description`, the `title` should be `Attempt to fix files` or something. The description should also note that this is an automatic operation.